pepper-apis/dss-core/src/test/java/org/broadinstitute/ddp/route/IntegrationTestSuite.java

```diff
@@ -136,11 +136,6 @@ public static void setup() {
     }
 
     public static void setup(boolean isCachingDisabled) {
-        if (callCounter > 0) {
```

Comment:
This code prevents proper initialization when we switch between the two methodologies we use to spin up database connections in tests.  We need to always assume we'll be running with tests with both so we can recover between approaches.

Reply:
InsertTestData does three things:
1) It establishes/re-establishes the connection pool.  Huzzah.  We need this.
2) InsertStaticTestData is used to ensure Liquibase consistency.  There doesn't appear to be any harm in revalidating the database structure.  It does add a bit of overhead/time - but it appears to be negligible. 
3) initializeStaticTestUserData is invoked.   If the test user were deleted, this would recreate it.  If the user is found, insertion code is skipped.